src/App.tsx

```diff
@@ -83,11 +71,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
```

Comment:
Replace with `posts.length`, it will be the same as `posts.length > 0`

Reply:
it will not. If I do like you suggest, there will be a '0' in the place of posts.length. You are not the first mentor in Mate that recommends this approach, but this is not the correct one. You can verify it from your end, plus I am attaching a screenshot to show this. No changes made, leaving as is
![Screenshot 2023-03-07 at 13 03 14](https://user-images.githubusercontent.com/57355180/223404392-864eb3f3-56e2-4fee-a733-b35d591d2f11.jpg)

To avoid the appearance of '0' I'd need to wrap this condition into Boolean(posts.length), but in this case I don't think this makes my code shorter & easier to read.

Reply:
Thank you for the code review, changes made and deployed